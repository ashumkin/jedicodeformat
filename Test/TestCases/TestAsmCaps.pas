unit TestAsmCaps;

{ AFS November 20077

  This code compiles, but is not semantically meaningfull.
  It is test cases for the code-formating utility

  Test capitalisation in asm
}

interface

implementation


var
  _NetGroupDel: Pointer;

procedure GetProcedureAddress(var P: Pointer; const ModuleName, ProcName: string);
begin
end;

procedure NetGroupDel;
begin  
  GetProcedureAddress(_NetGroupDel, 'Foo', 'bar');
  asm
           mov     ESP, EBP
           pop     EBP
           jmp     [_NetGroupDel]
  end;
end;

end.


