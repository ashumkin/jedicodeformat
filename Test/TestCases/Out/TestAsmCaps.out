unit TestAsmCaps;

{ AFS November 20077

  This code compiles, but is not semantically meaningfull.
  It is test cases for the code-formating utility

  Test capitalisation in asm
}

interface

implementation


var
  _NetGroupDel: Pointer;

procedure GetProcedureAddress(var P: Pointer; const ModuleName, ProcName: string);
begin
end;

procedure NetGroupDel;
begin
  GetProcedureAddress(_NetGroupDel, 'Foo', 'bar');
  asm
           MOV     ESP, EBP
           POP     EBP
           JMP     [_NetGroupDel]
  end;
end;

end.
